src/assets/files/questionnaire.js

```diff
@@ -514,8 +514,56 @@ export default {
                     }
                 ]
             },
-        ]
-    }, {
+            {
+                "linkId": "1.39",
+                "text": "Das ist eine Single-Choice-Abfrage als Drop-Down",
+                "type": "choice",
+                "required": true,
+                "answerOption": [{
+                        "valueString": "Option A"
+                    },
+                    {
+                        "valueString": "Option B"
+                    },
+                    {
+                        "valueString": "Option C"
+                    },
+                    {
+                        "valueString": "Option D"
+                    },
+                    {
+                        "valueString": "Option E"
+                    },
+                    {
+                        "valueString": "Option F"
+                    },
+                    {
+                        "valueString": "Option G"
+                    },
+                    {
+                        "valueString": "Option H"
+                    },
+                    {
+                        "valueString": "Option I"
+                    }
+                ],
+                "extension": [{
+                    "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
+                    "valueCodeableConcept": {
+                        "CodeableConcept": {
```

Comment:
This wrapping object is not valid FHIR. In FHIR, the `valueCodeableConcept` object directly contains `coding`

Reply:
This was fixed with https://github.com/NUMde/compass-numapp-frontend/commit/eead42f076e1d1021b24ef6296ec84c7e8c0e53e.

Reply:
Fixed with 04e90be. Thanks for the hint.

Reply:
Thank you for the quick fix :)